docs/developer-docs/setup/cycles/cycles-faucet.md

```diff
@@ -11,61 +11,60 @@ This guide explains how to use Cycles Faucet to acquire your first amount of 20T
 You will need to install Internet Computer SDK following [this guide](/developer-docs/setup/install/index.mdx).
 
 
-### Step 1: Authenticate
+### Step 1: Get Coupon
 
 First, you will need to navigate to <https://faucet.dfinity.org>. You
-will need to connect an active Twitter account to continue.
+will need to put in a request for cycles on the DFINITY DEV OFFICIAL Discord server. You can click on REQUEST CYCLES to join the discord server.
 
-![Connecting to Twitter](_attachments/faucet_step_1.png)
+![Getting Coupon](_attachments/faucet_step_1.png)
 
-If the Twitter account has not been used before, you are eligible to claim the 20T cycles.
+You must complete a survey sent to you by domwoe as a Direct Message. Once completed domwoe will send you a coupon.
```

Comment:
```suggestion
You must complete a survey sent to you by someone from the DFINTIY team as a Direct Message. Once completed, they will send you a coupon.
```
I'd prefer not calling out specific people by name. Also, if written generically, this line will go out of date less often.